### PR TITLE
Update block-point-to-point-file-transfers.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md [hunk 1]
```diff
@@ -67,10 +67,11 @@ To make this work, the user must be using a supported version of a 2016 Click-to
 	2. In the **Windows PowerShell** window, connect to your Microsoft 365 or Office 365 by running:
     
 		> [!NOTE]
-		> You only have to run the **Import-Module** command the first time you use the Skype for Business Online Windows PowerShell module.
+		> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md:70]:
```suggestion
		> The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

File: Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md [hunk 2]
```diff
@@ -67,10 +67,11 @@ To make this work, the user must be using a supported version of a 2016 Click-to
...
+		> If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md:71]:
```suggestion
	> If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md [hunk 3]
```diff
@@ -67,10 +67,11 @@ To make this work, the user must be using a supported version of a 2016 Click-to
 	2. In the **Windows PowerShell** window, connect to your Microsoft 365 or Office 365 by running:
     
 		> [!NOTE]
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/block-point-to-point-file-transfers.md:69]:
```suggestion
	> [!NOTE]
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**